tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -130,6 +136,24 @@ public function testEventPage(): void {
       trim($content_banner->filter('.oe-sc-event__oe-summary')->text())
     );
 
+    $date = $crawler->filter('dd');
```

Comment:
The selector appears bizarre to me.
It seems to work, but don't we have something better?